src/Courses/CourseDetailsTabs/TopicsTab/TopicsList.tsx

```diff
@@ -28,7 +29,7 @@ export const TopicsList: React.FC<TopicsListProps> = ({listOfTopics, flush, remo
         const topic = listOfTopics[rubric.source.index];
 
         return (
-            <ListGroupItem {...provided.draggableProps} {...provided.dragHandleProps} ref={provided.innerRef} variant={topic.errors > 0 ? 'danger' : undefined}>
+            <ListGroupItem {...provided.draggableProps} {...provided.dragHandleProps} ref={provided.innerRef} variant={(getUserRole() !== UserRole.STUDENT && topic.errors) > 0 ? 'danger' : undefined}>
```

Comment:
NIT: Weird parenthesis

You have:
```
(getUserRole() !== UserRole.STUDENT && topic.errors) > 0
```
But I think it should be
```
(getUserRole() !== UserRole.STUDENT && topic.errors > 0)
```

The first one is basically

(false && 5) > 0
false > 0 is false so it probably works 🤣